components/scream/cime_config/buildnml

```diff
@@ -215,6 +229,25 @@ def buildnml(case, caseroot, compname):
 
             fd.write("/\n")
 
+    # Create input data list
+    input_data_list_file = "{}/Buildconf/scream.input_data_list".format(caseroot)
```

Comment:
How is this file generated? I'm guessing CIME does it, but based on what info from our end?

Reply:
@bartgol , the code writing to this file is right below this line.

Reply:
LOL!